lib/DartEngine/docs/example2.cpp

```diff
@@ -0,0 +1,40 @@
+#include <DartEngine/Game.hpp>
+
+nlohmann::json save_settings = nlohmann::json::parse(R"({
```

Comment:
As I understand `R""` prefix provides to create a multiline string yes? Am I right?

Reply:
it's `R"()"` and you right.

Here is docs: https://en.cppreference.com/w/cpp/language/string_literal